src/components/ClaimReview/ClaimReviewForm.tsx

```diff
@@ -61,7 +64,8 @@ const ClaimReviewForm = ({
         isUnassigned ||
```

Comment:
issue(blocking): this is getting too complex, please consider moving this to a function and use the "guard clause pattern"